docs/source/pages/tutorials/provenance.rst

```diff
@@ -5,9 +5,11 @@
 Provenance discovery, extraction, and verification
 --------------------------------------------------
 
-This tutorial demonstrates how Macaron can automatically retrieve provenance for npm artifacts, validate the contents, and verify the authenticity. Any artifact that can be analyzed and checked for these properties can then be trusted to a greater degree than would be otherwise possible, as provenance files provide verifiable information, such as the commit and build service pipeline that has triggered the release.
+This tutorial demonstrates how Macaron can automatically retrieve provenance for artifacts, validate the contents, and verify the authenticity. Any artifact that can be analyzed and checked for these properties can then be trusted to a greater degree than would be otherwise possible, as provenance files provide verifiable information, such as the commit and build service pipeline that has triggered the release.
 
-For npm artifacts, Macaron makes use of available features provided by `npm <https://npmjs.com/>`_. Most importantly, npm allows developers to generate provenance files when publishing their artifacts. The `semver <https://www.npmjs.com/package/semver>`_ package is chosen as an example for this tutorial.
+Currently, Macaron supports discovery of attestation for: npm artifacts using features provided by `npm <https://npmjs.com/>`_; PyPI artifacts using features provided by `Open Source Insights <https://deps.dev/>`_; and artifacts that have been published attestations to `GitHub <https://docs.github.com/en/rest/repos/repos?apiVersion=2022-11-28#list-attestations>`_. This tutorial uses two example packages to demonstrate these three discovery methods: The `semver <https://www.npmjs.com/package/semver>`_ npm package, and the `toga <https://pypi.org/pypi/toga>`_ PyPI package.
```

Comment:
1. We also support provenances published as release assets on GitHub. Please add that as well. You can use [urllib3@2.0.0a1](https://github.com/oracle/macaron/tree/benmss/update-provenance-tutorial/tests/integration/cases/urllib3_expectation_file) which also has a CUE expectation.
2. You should also mention that we support provenance files if they’re provided as input, as long as they use one of the supported `predicateType` and `buildType` (see the full list here). For reference, witness provenance is one supported example. You can find more details in the command-line options documentation.
3. I think it would be clearer to list the supported ecosystems as bullet points for better readability.